tests/dummy/app/templates/application.hbs

```diff
@@ -1,6 +1,17 @@
 <h2>frost-css-core</h2>
 
-<h3>typography</h3>
+<h3>flex</h3>
```

Comment:
so technically core only has the flex demo right now, which i can split into another route for sure. I should take out the typography example. Typography isnt in core, its in theme.